packages/cordova/src/build.ts

```diff
@@ -68,7 +79,7 @@ const build : IBuild = (opts : ICordovaBuildOptions) => {
                                 },
                                 html: {
                                     replacements: {
-                                        injectScript: `<script src="/${wcFilename}"></script>`,
+                                        injectScript: `<script>${wcScriptData}</script>`,
```

Comment:
The webengine will be more efficient if it loads the file as src instead of a blob of JS I think